params.go

```diff
@@ -181,6 +181,12 @@ func IntToPointer(i int) *int {
 	return &i
 }
 
+// StringToPointer is a function for converting strings into string pointers.
+// This is useful when passing in options to operations.
+func StringToPointer(s string) *string {
```

Comment:
> Unless I'm mistaken, one difference is that StringToPointer will return &"", where MaybeString will not.

Yes, I know :) I don't think we need this function. Creating a pointer to a `string` (as evidenced by this function) is trivial, and I suspect creating one for a `string` constant is uncommon. If you'd like something for the tests, you can define a exported variable in one of the common testing packages:
```go
var(
  Empty = ""
  EmptyPtr = &empty
)
```